### DISCUSSION TITLE
WIN11 22H2安装包更新，已解决任务栏错位问题。

### DISCUSSION THREAD

**Discussion by lovepepper** [zhongyang219/TrafficMonitor#1322] [Q&A]
Dear all，
开发者已经更新了，只不过还没有release安装包，我打包一下供大家下载。
如果有任何涉及开发者个人利益问题，请告知我，我删除链接。

百度云：链接：https://pan.baidu.com/s/1pdW9WzBLyJxQTLkKNn4jfg?pwd=7666 
提取码：7666
阿里云：https://www.aliyundrive.com/s/M9Fj1TBn2cH

阿里云因为云盘限制，上传的文件经过转换，需要使用winRAR或者7zip等解压软件打开，不可使用WIN11资源管理器直接打开。

**Comment by xzl0326** [comment]
无法运行你这个版本，会报错缺少dll

**Reply by lovepepper** [reply]
> 无法运行你这个版本，会报错缺少dll

我确认可以正常使用，解压至任意目录，双击.exe即。
也可仅用上述包中的.exe替换你PC中的.exe，一样可以使用。

**Reply by findwww** [reply]
相同的问题，提示找不到mfc140ud.dll，替换也没用

**Reply by lovepepper** [reply]
我尝试查找该文件，在下面路径下确有此文件。
C:\Windows\System32
C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Redist\MSVC\14.32.31326\debug_nonredist\x64\Microsoft.VC143.DebugMFC
C:\Windows\SysWOW64
推测是你们PC缺少微软运行库或者其他组件。
[mfc140ud.zip](https://github.com/zhongyang219/TrafficMonitor/files/9690120/mfc140ud.zip)
我将此文件附上，你放在第一个路径下看是否能正常使用，如不能，请查找并安装必要组件。

**Reply by lovepepper** [reply]
更新了安装包，解决以上问题

**Comment by lovepepper** [comment]
TrafficMonitor_V1.83_x64.zip: https://url23.ctfile.com/f/13898623-685829295-3e4499?p=2935 （访问密码：2935）


slow, but no register

**Reply by room216** [reply]
Hello, I recently upgraded to Windows 11 22H2 and ran into the issue with Traffic Monitor on the taskbar overlapping the icons in the taskbar.

I downloaded your patch file here and replaced it with the current files. But I cant get it to work because multiple pop up of missing .dll files show up when I tried to run "TrafficMonitor.exe"

Any help please?

**Reply by lovepepper** [reply]
i have updated the zip, pls Redownload.

**Comment by xzl0326** [comment]
我直接下载Actions里的版本了。那个可以正常使用。应该和你打包的环境有关系
